mmv1/templates/terraform/examples/alloydb_secondary_instance_basic.tf.erb

```diff
@@ -32,6 +32,10 @@ resource "google_alloydb_cluster" "secondary" {
 
   deletion_policy = "FORCE"
 
+  lifecycle {
+    ignore_changes = [instance_type]
+  }
```

Comment:
Note: Need `lifecycle.ignore_changes` because `instance_type` is an `immutable` field, and when promoting cluster from `SECONDARY` to `PRIMARY`, the instance_type of the associated secondary instance also changes and becomes PRIMARY. And we do not want terraform to destroy and create the instance because the field is immurable

Reply:
Can we comment why we mark it this way inline?

Reply:
Added inline comment.